src/transformers/models/bart/modeling_tf_bart.py

```diff
@@ -1041,25 +1147,26 @@ def resize_token_embeddings(self, new_num_tokens):
             num_tokens_to_copy = min(self.final_logits_bias.shape[0], new_num_tokens)
             init_bias = tf.zeros((new_num_tokens,))
             init_bias[:num_tokens_to_copy] = self.final_logits_bias.value()[:num_tokens_to_copy]
-            name = self.name + "/final_logits_bias"
             self.final_logits_bias = self.add_weight(
                 shape=(1, new_num_tokens),
                 initializer="zeros",
                 trainable=False,
-                name=name,
+                name="final_logits_bias",
```

Comment:
This changes from `name/final_logits_bias` to simply `final_logits_bias`, it that ok?

Reply:
Under the hood, for TF this is the same.